VPet-Simulator.Windows/mod/0000_core/lang/en/Base2306.lps

```diff
@@ -67,7 +67,7 @@ Y轴#Y-axis:|
 使用桌宠开发者\r提供的免费API#Use free VPET developer\rprovided API:|
 需遵循相关协议法律法规并有聊天字数限制#Applicable agreements, laws and regulations apply, and chat word limits apply:|
 初始化桌宠聊天程序#Initialize VPET chat program:|
-互动#Interact:|
+互动#Actions:|
```

Comment:
I think that this should be `Interactions`

Reply:
I think keep `Interact` is ok because this is the menu bar, the space is very small, and there is no way to insert more words